MyoSimulatorForm/MyoSimulatorForm/HubCommunicator.cs

```diff
@@ -44,7 +44,7 @@ public enum Pose
             WAVE_IN,
             WAVE_OUT,
             FINGERS_SPREAD,
-            RESERVED1,
+            RESERVED1,      /* KEEP THIS! Must match the hub's pose enum */
             THUMB_TO_PINKY,
             UNKNOWN
         }
```

Comment:
I wonder if we can have an assert that the versino of Myo SDK we're using is 5.1, so that if its NOT that version, this code will fail, but through assertion so we know what's wrong. Just some forethought assuming we upgrade to the latest SDK.